lambda/src/main/scala/pricemigrationengine/handlers/EstimationHandler.scala

```diff
@@ -80,18 +81,26 @@ object EstimationHandler extends CohortHandler {
   /*
    * Earliest start date spread out over 3 months.
    */
-  private[handlers] def spreadEarliestStartDate(
+  def spreadEarliestStartDate(
       subscription: ZuoraSubscription,
       invoicePreview: ZuoraInvoiceList,
       earliestStartDate: LocalDate
   ): IO[ConfigFailure, LocalDate] = {
 
+    def relu(number: Int): Int =
+      if (number < 0) 0 else number
+
     lazy val earliestStartDateForAMonthlySub =
       for {
         yearAgo <- Clock.localDateTime.map(_.toLocalDate.minusYears(1))
         randomFactor <-
           if (subscription.customerAcceptanceDate.isBefore(yearAgo)) Random.nextIntBetween(0, 3)
-          else Random.nextIntBetween(12, 15)
+          else
+            ZIO.succeed(
+              relu(
+                ChronoUnit.MONTHS.between(earliestStartDate, subscription.customerAcceptanceDate.plusMonths(13)).toInt
+              )
+            )
```

Comment:
Maybe move this block into its own function with some commentary as it's not immediately obvious what the intention is here